spec/presenters/hyku/manifest_enabled_work_show_presenter_spec.rb

```diff
@@ -57,4 +57,29 @@
       expect(subject[0]['value']).to include('Test title', 'Another test title')
     end
   end
+
+  context "when the work has identifier" do
+    let(:document) do
+      { "identifier_tesim" => %w[
+        ISBN:978-83-7659-303-6 978-3-540-49698-4 9790879392788
+        doi:10.1038/nphys1170 3-921099-34-X 3-540-49698-x 0-19-852663-6
+      ] }
+    end
+
+    describe "#doi" do
+      it "extracts the DOI from the identifiers" do
+        # puts solr_document.inspect
+        expect(presenter.doi).to eq("10.1038/nphys1170")
+      end
+    end
+
+    describe "#isbns" do
+      it "extracts ISBNs from the identifiers" do
+        expect(presenter.isbns).to eq(%w[
+                                        978-83-7659-303-6 978-3-540-49698-4 9790879392788
+                                        3-921099-34-X 3-540-49698-x 0-19-852663-6
+                                      ])
+      end
+    end
+  end
```

Comment:
Can you please add some contexts for when the work doesn't have an identifier and when the identifier doesn't match the two queries?